make/source.mk

```diff
@@ -61,6 +61,7 @@ COMMON_SRC = \
             io/transponder_ir.c \
             msp/msp_serial.c \
             pg/adc.c \
+			pg/flash.c \
```

Comment:
Indentation 😉

Reply:
Dammit, my vim seems to be using tabs in .mk files for some reason.